package.json

```diff
@@ -9,7 +9,7 @@
     "coverage": "nyc mocha",
     "start": "ts-node -r tsconfig-paths/register app/server.ts",
     "start-dev": "strict-npm-engines && ts-node-dev --no-notify -r tsconfig-paths/register --watch app/views --respawn app/server",
-    "start-dev-fast": "TS_NODE_TRANSPILE_ONLY=true ts-node-dev --no-notify -r tsconfig-paths/register --respawn app/server",
+    "start-dev-fast": "TS_NODE_TRANSPILE_ONLY=true ts-node-dev --no-notify -r tsconfig-paths/register --respawn  --respawn --inspect=127.0.0.1:9229 app/server",
```

Comment:
What's the difference between this and the `watch-debug` command? Can we get rid of the `nodemon` dev dependency we use for watch-debug?

Reply:
This lets me start the debugger on a specific port (don't know if that is possible with watch-debug) so I can debug harmony, service runner, and query-cmr all from one vscode session (each gets its own debug port).